_includes/pbc/all/install-features/202404.0/install-the-merchant-feature.md

```diff
@@ -0,0 +1,810 @@
+
+
+This document describes how to install the Merchant feature.
+
+## Install feature core
+
+Follow the steps below to install the Merchant feature.
+
+## Prerequisites
+
+Install the required features:
+
+| NAME | VERSION | INSTALLATION GUIDE |
+|-|-|-|
+| Spryker Core | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
+
+## 1) Install the required modules
+
+Install the required modules using Composer:
+
+```bash
+composer require spryker-feature/merchant
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE                  | EXPECTED DIRECTORY                       |
+|-------------------------|------------------------------------------|
+| Merchant                | spryker/merchant                         |
```

Comment:
* We have to add plugin from this module to the guide, because they are only exist in the migration guides.